python/setup.py

```diff
@@ -436,19 +436,28 @@ def copy_libs(dir):
             else:
                 build_prefix = self.build_type
 
+            # Copy PyArrow headers to pyarrow/include
```

Comment:
Could we also do this after the `if self.bundle_arrow_cpp or self.bundle_arrow_cpp_headers:`, so we don't have to repeat it there?

Reply:
🤦‍♀️ Yes, of course =)